RELEASE_NOTES/2.23.0.md

```diff
@@ -0,0 +1,140 @@
+
+`SharedTree` now takes advantage of a new feature called "op bunching" where contiguous ops in a grouped batch are bunched and processed together. This improves the performance of processing ops asymptotically; as the number of local ops and incoming ops increase, the processing time will reduce.
+
+For example, with 10 local ops + 10 incoming ops, the performance increases by 70%; with 100 local ops + 100 incoming ops, the performance increases by 94%.
+
+This will help improve performance in the following scenarios:
+
+- A client makes a large number of changes in a single JS turn. For example, copy pasting large data like a table.
+- A client has a large number of local changes. For example, slow clients whose changes are slow to ack or clients with a local branch with large number of changes.
+
+#### Change details
+
+Commit: [`a98b04f`](https://github.com/microsoft/FluidFramework/commit/a98b04fc9e000971bdfa8135251a7dc3e189502c)
+
+Affected packages:
+
+- @fluidframework/tree
+- fluid-framework
+
+[⬆️ Table of contents](#contents)
+
+### Invalid schema base classes in Tree.is now throw an error instead of returning false ([#23938](https://github.com/microsoft/FluidFramework/issues/23938))
+
+As documented in [`TreeNodeSchemaClass`](https://fluidframework.com/docs/api/fluid-framework/treenodeschemaclass-typealias#treenodeschemaclass-remarks), there are specific rules around sub-classing schema, mainly that only a single most derived class can be used. One place where it was easy to accidentally violate this rule and get hard-to-debug results was [`Tree.is`](https://fluidframework.com/docs/data-structures/tree/nodes#treeis). This has been mitigated by adding a check in `Tree.is` which detects this mistake (which used to result in `false` being returned) and instead throws a `UsageError` explaining the situation. The error will look something like:
+
+> Two schema classes were used (CustomObjectNode and Derived) which derived from the same SchemaFactory generated class ("com.example.Test"). This is invalid.
+
+For applications wanting to test if a given `TreeNode` is an instance of some schema base class, this can be done using `instanceof` which includes base base classes when doing the check.
+
+#### Change details
+
+Commit: [`0099565`](https://github.com/microsoft/FluidFramework/commit/00995654070a4e13b57b2562ff4a5935aba70a2f)
+
+Affected packages:
+
+- @fluidframework/tree
+- fluid-framework
+
+[⬆️ Table of contents](#contents)
+
+### Creating large transactions and processing inbound changes is now faster ([#23929](https://github.com/microsoft/FluidFramework/issues/23929))
+
+SharedTree sometimes composes several sequential changes into a single change. It does so whenever a transaction is created and when processing inbound changes.
+
+Version 2.3.0 makes this composition process asymptotically faster. For example, creating a transaction that performs 1000 edits on a single array now takes 170ms instead of 1.5s (an 89% improvement).
```

Comment:
I think this was the intention?
```suggestion
Version 2.23.0 makes this composition process asymptotically faster. For example, creating a transaction that performs 1000 edits on a single array now takes 170ms instead of 1.5s (an 89% improvement).
```